### DISCUSSION TITLE
啥时候有极简首页

### DISCUSSION THREAD

**Discussion by ChaXxl** [the1812/Bilibili-Evolved#2620] [功能建议 / Ideas]
![image-20211208225413475](https://s2.loli.net/2021/12/08/GBu49Vlxve2btAc.png)

<span style="font-size: 42px; color: azure; ">极简前：</span>
![image-20211208225009968](https://s2.loli.net/2021/12/08/MAwH6NZShtFdDYj.png)

<span style="font-size: 42px; color: azure; ">极简后，还是这个舒服呀</span>
![image-20211208225105753](https://s2.loli.net/2021/12/08/lIQpdvrDky2FnEa.png)

**Comment by the1812** [comment]
等 v2 发完正式, 并且 b 站别老改样式, 我就要开始弄了
